build.rs

```diff
@@ -77,6 +77,7 @@ fn compile_sass() -> Result<(), Box<dyn Error>> {
 
     // Compile rustdoc.scss -> rustdoc.css
     compile_sass_file("rustdoc", "rustdoc", &[])?;
+    compile_sass_file("rustdoc-2", "rustdoc-2", &[])?;
```

Comment:
This isn't super informative - could you name it after the date or PR number instead?

Reply:
Done.